ApiDoctor.Validation/Tags/TagProcessor.cs

```diff
@@ -39,10 +39,15 @@ public class TagProcessor
         private string[] TagsToInclude = null;
         private string DocSetRoot = null;
         private static string[] tagSeparators = { ",", " " };
+        private static string[] docFxAlerts = new[] { "NOTE", "TIP", "IMPORTANT", "CAUTION", "WARNING" };
```

Comment:
what's all this docfx-specific stuff for?

Reply:
https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=master

Docfx syntax allows the content writers to put things such as alerts, videos etc, into the documentation. As such ApiDoctor needs to skip validating such content lines since they look similar to links.

Reply:
understood.